source/usage-examples/findOne.go

```diff
@@ -4,43 +4,49 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
-	"log"
 )
 
-func main() {
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+
+var ctx = context.TODO()
 
-	// Replace the uri string with your MongoDB deployment's connection string.
-	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
-	ctx := context.TODO()
+func main() {
 	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
 	if err != nil {
 		panic(err)
 	}
-	// Disconnect the client once the function returns.
+
 	defer func() {
 		if err = client.Disconnect(ctx); err != nil {
 			panic(err)
 		}
+
 	}()
+
 	coll := client.Database("sample_mflix").Collection("movies")
 	var result bson.D
-	opts := options.FindOne()
-	opts.SetProjection(bson.D{{"_id", 0}, {"title", 1}, {"imdb", 1}})
-	opts.SetSort(bson.D{{"imdb.rating", -1}})
+	opts := options.FindOne().
+		SetProjection(bson.D{{"_id", 0}, {"title", 1}, {"imdb", 1}}).
+		SetSort(bson.D{{"imdb.rating", -1}})
 	err = coll.FindOne(ctx, bson.D{{"title", "The Room"}}, opts).Decode(&result)
 	if err != nil {
 		if err == mongo.ErrNoDocuments {
 			// This error means your query did not match any documents.
 			return
 		}
-		log.Fatal(err)
+
```

Comment:
And here, this newline can be removed.

Reply:
Comment: 
Fix: https://github.com/mongodb/docs-golang/pull/2/commits/72cf9221c25b39c390a7d7162a0c64c03f75c549